src/Box/Box.cs

```diff
@@ -104,8 +106,13 @@ void OnSimulationEnded()
 			
 			rigidBody.LinearVelocity = Vector3.Zero;
 			rigidBody.AngularVelocity = Vector3.Zero;
-			
-			GlobalPosition = initialPos;
+
+			//Work around for #83 
+			if (enable_inital_pos)
+			{
+				GlobalPosition = initialPos;
+			}
```

Comment:
This prevents the warp to zero, but leaves the box out of position. It will use that new position as initialPos next run, so we still the same sort of problem. Have you considered exporting initialPos, but hiding it in the editor?

Reply:
I'm not sure I understand. It will be in the position it was when the scene was last saved. On the next run it will store the initalPos correctly OnSimulationStarted.

Reply:
This PR is meant to prevent scenes from being modified outside user interaction and saved accidentally, which is the main issue, not necessarily to recover some kind "in the middle of a simulation" state.

Reply:
> It will be in the position it was when the scene was last saved.

This wasn't true in my testing. It looks like the box will be in the position it was at when the "Stop" button was clicked for the first time after the reload.

The steps I used are here, but the result has changed: https://github.com/Open-Industry-Project/Open-Industry-Project/issues/83#issuecomment-2442433863

Reply:
Should be good now.